makedist

```diff
@@ -297,32 +331,47 @@ if [ "$ARCH" = "ppc64le" ]; then
     CVMFSRPMNAME="$(basename $CVMFSRPMURL)"
     CVMFSVERSION="`echo "$CVMFSRPMNAME" 's/.*cvmfs-\([^-]*\)-.*/\1/'`"
 else
-    # Now that we can figure out the latest cvmfs version, download the
-    #   corresponding cvmfs packages from CERN instead
-    CVMFSVERSION="`echo "$CVMFSPKG"|sed 's/.*cvmfs-\([^-]*\)-.*/\1/'`"
-    CVMFSURL="https://ecsft.cern.ch/dist/cvmfs/cvmfs-$CVMFSVERSION"
-    CVMFSRPMNAME="`curl -Ls $CVMFSURL/|grep "cvmfs-$CVMFSVERSION-.*$MACH.*\.$ARCH"|sed 's/.*href="//;s/".*//'|tail -1`"
-    if [ -z "$CVMFSRPMNAME" ]; then
-        echo "No matching cvmfs rpm found at $CVMFSURL" >&2
+    # Get  CVMFS package download urls from CERN yum repo ( with possible fallback )
+    DISTRO_YUM="`distroname_yum $DISTRO`"
+    CVMFSURL="$CVMFS_BASEURL/yum/cvmfs/$DISTRO_YUM/$VERS/$ARCH"
+    # for the listing, we need to query the "s3-website" variant of the s3 url
+    CVMFS_BASELIST="$(get_cvmfs_pgk_list ${CVMFSURL/cvmrepo.s3/cvmrepo.s3-website})"
+    CVMFSVERSION="$(get_cvmfs_latest_version_from_pkg_list $CVMFS_BASELIST)"
+    if [ -z "CVMFSVERSION" ]; then
+      echo "WARN: Couldn't find package listing in mirror $CVMFSURL, switching over.." >&2
+      CVMFS_BASEURL="$CVMFS_BASE_URL_MIRROR2"
+      CVMFSURL="$CVMFS_BASEURL/yum/cvmfs/$DISTRO_YUM/$VERS/$ARCH"
+      CVMFS_BASELIST="$(get_cvmfs_pgk_list $CVMFSURL)"
+      CVMFSVERSION="$(get_cvmfs_latest_version_from_pkg_list $CVMFS_BASELIST)"
+      if [ -z "CVMFSVERSION" ]; then
+        echo "ERROR: Couldn't find package listing in mirror $CVMFS_BASEURL, and no mirrors left! " >&2
         exit 1
+      fi
     fi
+    CVMFSRPMNAME="cvmfs-$CVMFSVERSION-1.$MACH.$ARCH.rpm"
     CVMFSRPMURL="$CVMFSURL/$CVMFSRPMNAME"
 fi
-URLS="$CVMFSRPMURL"
 
+CVMFSRPMURL_LIBS=""
 MINORVERSION="`echo "$CVMFSVERSION"|cut -d. -f2`"
 if [ "$MINORVERSION" -ge 10 ]; then
     # include cvmfs-libs
-    URLS="$URLS $CVMFSURL/`echo $CVMFSRPMNAME|sed 's/cvmfs-/cvmfs-libs-/'`"
+    CVMFSRPMURL_LIBS="$CVMFSURL/`echo $CVMFSRPMNAME|sed 's/cvmfs-/cvmfs-libs-/'`"
 fi
 
+URLS="$URLS $CVMFSPRMPURL $CVMFSRPMURL_LIBS"
```

Comment:
Here's the typo causing the ci test failures.  `CVMFSPRMPURL` -> `CVMFSRPMURL`

Reply:
```suggestion
URLS="$URLS $CVMFSRPMPURL $CVMFSRPMURL_LIBS"
```

Reply:
Ah, I just fixed 1/2 typos. Sorry, I'm out of office at the moment.